README.md

```diff
@@ -125,6 +130,15 @@ Activation will backup your $PATH, prepend it with the installation's bin/
 directory. Thus it's only valid for the current shell session, and until you
 activate another installation or call `kerl_deactivate`.
 
+**Note**: alternatively you can use `kerl build-install` as a shortcut for
+the two previous actions to be played in sequence.
+
+    $ kerl build-install
+    usage: ./kerl build-install <release> [build_name] [directory]
+
+    $ kerl build-install git
+    usage: ./kerl build-install git <git_url> <git_version> [build_name] [directory]
+
```

Comment:
This is misplaced (this pull request was born in the other branch); will remove.

Reply:
It was removed in a recent commit.